File: test/py/case70_class_instance.py
class Box70:
    def __init__(self, seed: int) -> None:
        self.seed = seed

    def next(self) -> int:
        self.seed = self.seed + 1
        return self.seed


if __name__ == "__main__":
    b: Box70 = Box70(3)
    print(b.next())

File: test/cs/case70_class_instance.cs
using System;

public static class Program
{
    public class Box70
    {
        public int seed;

        public Box70(int seed)
        {
            this.seed = seed;
        }
        public int next()
        {
            this.seed = (this.seed + 1);
            return this.seed;
        }
    }

    public static void Main(string[] args)
    {
        Box70 b = new Box70(3);
        Console.WriteLine(b.next());
    }
}
